src/pages/SalesPage/SaleSummaryCard.tsx

```diff
@@ -16,10 +17,17 @@ const SaleSummaryCard: React.VFC = () => {
 
   return (
     <>
-      {!isAuthenticating && !permissions.GROSS_SALES_ADMIN && !permissions.GROSS_SALES_NORMAL ? <UnAuthCover /> : null}
+      {!isAuthenticating &&
+      !permissions.GROSS_SALES_ADMIN &&
+      !permissions.GROSS_SALES_NORMAL &&
+      !permissions.READ_GROUP_SALES_ALL ? (
+        <UnAuthCover />
+      ) : null}
       <AdminCard>
         {!isAuthenticating && appId && permissions.GROSS_SALES_ADMIN ? (
           <SalesSummaryAdminCard appId={appId} />
+        ) : !isAuthenticating && appId && memberId && permissions.READ_GROUP_SALES_ALL ? (
+          <SalesSummaryGroupCard memberId={memberId} />
         ) : !isAuthenticating && appId && memberId && permissions.GROSS_SALES_NORMAL ? (
           <SalesSummaryNormalCard memberId={memberId} appId={appId} />
         ) : (
```

Comment:
`!isAuthenticating && appId && memberId`
 could simplify this code by extracting the logic into a separate function